app/views/content/life-as-a-teacher/pay-and-benefits/abigails-favourite-things-about-teaching/_article.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <p>The best thing about teaching is having a positive impact on pupils’ school experience by helping them to understand maths and enjoy learning about it too! I also enjoy being a form tutor and looking out for pupils’ wellbeing.</p>
+  </section>
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "Building good relationships with pupils so they are keen to learn and keep trying, even when they find topics challenging, is really rewarding.",
+    large: true
+  ) %>
+  </section>
+    <section class="col col-720 col-space-l-top">
```

Comment:
This <section> and its contents should be outdented to line up with the closing </section> tag